prime-router/src/main/kotlin/cli/tests/AuthTests.kt

```diff
@@ -528,10 +528,9 @@ class Server2ServerAuthTests : CoolTest() {
         val baseUrl = environment.url.toString() + HttpUtilities.tokenApi
         val privateKey = SenderUtils.readPrivateKeyPem(privateKeyStr)
         val senderSignedJWT = SenderUtils.generateSenderToken(sender, baseUrl, privateKey, kid)
-        val senderTokenUrl =
-            SenderUtils.generateSenderUrl(settingsEnv, senderSignedJWT, scope)
-
-        return HttpUtilities.postHttp(senderTokenUrl.toString(), "".toByteArray())
+        val senderTokenUrl = settingsEnv.formUrl("api/token").toString()
+        val body = SenderUtils.generateSenderUrlParameterString(senderSignedJWT, scope)
+        return HttpUtilities.postHttp(senderTokenUrl, body.toByteArray())
```

Comment:
Note line 533 now sends the parameters in the body.

Reply:
See my comment in the token.yml about making sure the response content type (header) matches the format of the data returned. 

Some client libraries will throw exceptions if the content type is `application/json` but the body isn't valid json.

IMO: REST API should always return json and put content into fields even if it's just a single value. The helps "future proof" the client code because it generically parses json then just pulls out the fields it understands.